javascript-this/javascript-this-quiz-notes.md

```diff
@@ -0,0 +1,73 @@
+# javascript-this-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is `this` in JavaScript?
+this is an implicit parameter of all JavaScript functions
```

Comment:
True, this is implicit but there is a more specific answer to what `this` is. Take a shot again after Q&A :)